apps/frontend/src/components/global/sidebaritems/SidebarFileList.vue

```diff
@@ -116,17 +118,16 @@ export default class FileItem extends mixins(ServerMixin, RouteMixin) {
     }
     axios
       .post('/evaluations', formData)
-      .then((response) => {
+      .then(() => {
         SnackbarModule.notify('File saved successfully');
-        file.database_id = parseInt(response.data.id);
```

Comment:
How do we get the ID of the file if it is no longer stored here?

Reply:
This was originally to prevent the user from editing their local copy of the uploaded profile as it would should empty fields when you tried to edit the thing you just uploaded, however, adding the created IEvaluation just lets the user edit it after saving.